spec/features/assignment_creation_spec.rb

```diff
@@ -588,29 +557,6 @@ def fill_assignment_form
       fill_in 'late_policy_max_penalty', with: 2
       click_button 'Create'
     end
-
-    # able to set deadlines for a single round of reviews
-    it "set the deadline for an assignment review" do
```

Comment:
I just afraid the code may not that readable for later developers. But you did a great job! And there may be a tradeoff between readable and duplicate code.